xml/obs/openSUSE:Leap:15:ARM.xml

```diff
@@ -1,11 +1,11 @@
 <openQA
-    project_pattern="openSUSE:Leap:(?P&lt;version&gt;15.2):ARM:ToTest"
+    project_pattern="openSUSE:Leap:(?P&lt;version&gt;15.[3-9]):ARM:ToTest"
     dist_path="images/local"
-    archs="aarch64">
-    <flavor name="NET|DVD" folder="*product*" distri="opensuse" iso="1">
+    archs="armv7hl">
+    <flavor name="NET" folder="*product*" distri="opensuse" iso="1">
         <repos>
-            <oss folder="openSUSE-ftp-ftp" debug="{java*,kernel-default-debug*,kernel-default-base-debug*,mraa-debug*}" source="{coreutils*,yast2-network*}" mirror="1" dest="openSUSE-15.2-oss-aarch64"/>
+            <oss folder="Leap-ftp-ftp" debug="{java*,kernel-default-debug*,kernel-default-base-debug*,mraa-debug*}" source="{coreutils*,yast2-network*}" mirror="1" dest="openSUSE-Leap-${version}-oss-$arch"/>
         </repos>
     </flavor>
-    <news iso="DVD" archs="aarch64"/>
+<!--     <news iso="DVD" archs="armv7hl"/> -->
```

Comment:
Leftover?

Reply:
I left it to think about it on next update.